### DISCUSSION TITLE
error when clicking on messages in Osu!Lazer

### DISCUSSION THREAD

**Discussion by HydroZenon** [ppy/osu#17288] [Q&A] [answered]
![Screenshot (143)](https://user-images.githubusercontent.com/101785393/158739843-ddee9570-0bfb-4133-952b-2a59ee402291.png)
This is pretty self-explanatory. This is a pretty light issue, but it leads to my game crashing. Thanks in advance!

**Comment by smoogipoo** [comment]
![image](https://user-images.githubusercontent.com/1329837/158740034-81fd80bf-d558-43f1-a8f1-e7c2a8d63315.png)
https://github.com/ppy/osu/discussions/15188

**Reply by HydroZenon** [reply]
how?

**Reply by smoogipoo** [reply]
Everything is explained in the thread I linked... https://github.com/ppy/osu/discussions/15188

**Reply by HydroZenon** [reply]
sorry man first time using github!

**Reply by HydroZenon** [reply]
which logs folder is it
![Screenshot (145)](https://user-images.githubusercontent.com/101785393/158740430-44665d7b-54f3-43cc-aaab-732897e17902.png)
?

**Reply by smoogipoo** [reply]
You're in the correct folder, drag-drop all of those files you see there.

**Reply by HydroZenon** [reply]
okay
[network.log](https://github.com/ppy/osu/files/8281521/network.log)
[performance.log](https://github.com/ppy/osu/files/8281522/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8281523/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8281524/updater.log)
[database.log](https://github.com/ppy/osu/files/8281525/database.log)

**Comment by frenzibyte** [comment, marked as answer]
Looks to be an oversight in the code, will take this one.

**Reply by HydroZenon** [reply]
alright tell me when resolved so I can mark the post as is.

**Reply by bdach** [reply]
we will resolve the discussion when it's fixed.

**Reply by HydroZenon** [reply]
oh, okay I didn't know you can do that! sorry to sound like an ass

**Reply by smoogipoo** [reply]
Thanks, escalated to https://github.com/ppy/osu/issues/17289
